aten/src/ATen/function_wrapper.py

```diff
@@ -215,6 +216,7 @@ def __init__(self, reason):
     'THTensor*': 'Tensor',
     'THSTensor*': 'SparseTensorRef',
     'THByteTensor*': 'BoolTensor',
+    'THBoolTensor*': 'BoolTensor',
```

Comment:
wait both Bool and Byte map to BoolTensor?  How do you check you have the properly dtype at the TH boundary then?